File: sources/julia/07_string_append.jl
# task 07 string_append — expected output: 1000000
# build: julia 07_string_append.jl    run: julia 07_string_append.jl

function main()
    text = ""
    for _ in 1:1000000
        text = text * "x"
    end
    println(length(text))
end

main()

File: sources/javascript/07_string_append.js
// task 07 string_append — expected output: 1000000
// build: none    run: node 07_string_append.js | bun 07_string_append.js | deno run 07_string_append.js

let text = '';
for (let i = 0; i < 1000000; i++) {
  text = text + 'x';
}
console.log(text.length);
